plotly-plot-behavior.js

```diff
@@ -189,6 +189,17 @@
         return self.fire('plotly-unhover', {data: data});
       };
 
+      /**
+       * Custom plotly-specific event for binding to zoom events.
+       * Fires after zoom.
+       *
+       * @event plotly_relayout
+       * @see the {@link https://plot.ly/javascript/zoom-events/}
+       */
+      self._onPlotlyRelayout = function (data) {
+        return self.fire('plotly_relayout', {data: data});
```

Comment:
The name of the fired event should be `plotly-relayout`, because Polymer event naming conventions use dashes. (See the above events).